frontend/src/pages/Board/Board.js

```diff
@@ -104,15 +107,16 @@ const Board = () => {
 
   // @ jenn here's rest of info you need!
   function discoverBoard(board) {
-    console.log("DISCOVER BOARD: ");
-    console.log("board id: ", board.id);
-    console.log("user id: ", board.userId);
-    console.log("-----------------------")
+    const url = `/boards/${board.userId}/${board.id}`;
+    window.open(url, '_blank');
```

Comment:
when the user opens up the new board from discover, is it possible for the "go back" button to route back to the user's original board for that book? currently it routes back to the user's general boards